es5-shim.js

```diff
@@ -329,6 +360,97 @@ var strSplit = call.bind(StringPrototype.split);
 var strIndexOf = call.bind(StringPrototype.indexOf);
 var push = call.bind(array_push);
 
+// ES-5 15.3.4.4
+// http://es5.github.io/#x15.3.4.4
+var hasToStringTagBasicBug = !(toStr() === '[object Undefined]' && toStr(null) === '[object Null]');
+var hasToStringTagArgumentsBug = toStr(function () {
+  return arguments;
+}) !== '[object Arguments]';
+var hasToStringTagArrayBug = toStr([]) !== '[object Array]';
+var hasToStringTagRegExpBug = toStr(/test/) !== '[object RegExp]';
```

Comment:
IE6-8 -> hasToStringTagBasicBug
The others I can't remember exactly, but could pick the tests across and run them as I have no way of testing any more.